tests/integration/dump/test_dump_dbms.py

```diff
@@ -149,15 +149,31 @@ def error_callback(resp):
 
     dump_path = os.path.join(str(tmpdir), 'dump_dir')
     os.environ['DBMS_WORKSPACE'] = os.path.join(str(tmpdir), 'index_ws')
+    os.environ['QUERY_WORKSPACE'] = os.path.join(str(tmpdir), 'query_ws')
+
     with Flow.load_config('flow_dbms.yml') as flow_dbms:
-        with TimeContext(f'### indexing {len(docs)} docs'):
-            flow_dbms.index(docs)
+        with Flow.load_config('flow_query.yml') as flow_query:
+            with TimeContext(f'### indexing {len(docs)} docs'):
+                flow_dbms.index(docs)
+
+            with TimeContext(f'### dumping {len(docs)} docs'):
+                flow_dbms.dump('indexer_dbms', dump_path, shards=shards, timeout=-1)
 
-        with TimeContext(f'### dumping {len(docs)} docs'):
-            flow_dbms.dump('indexer_dbms', dump_path, shards=shards, timeout=-1)
+            dir_size = path_size(dump_path)
+            print(f'### dump path size: {dir_size} MBs')
+
+            # TODO add param to dump path
+            flow_query.rolling_update('indexer_query')
```

Comment:
In order to pass a parameter, we need to refactor how `executors` are being reloaded at __init__ time.